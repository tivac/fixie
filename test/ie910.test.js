"use strict";

var simple = require("./util.js").simple,
    nested = require("./util.js").nested;

describe(":ie910", () => {
    it("should transform :ie910(...) into an ie910 (single selector)", () =>
        simple(`
            :ie910(.fooga) {
                color: red;
            }
        `)
    );

    it("should transform :ie910(...) into an ie910 (multiple selectors)", () =>
        simple(`
            :ie910(.fooga .booga) {
                color: red;
            }
        `)
    );

    it("should work with postcss-nested", () =>
        nested(`
            .fooga {
                color: red;
                
                :ie910(&) {
                    color: blue;
                }
            }
        `)
    );
});
