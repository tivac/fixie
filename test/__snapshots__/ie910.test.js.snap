// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`:ie910 should transform :ie910(...) into an ie910 (multiple selectors) 1`] = `
"@media screen and (min-width:0\\\\0) {.fooga .booga {
    color: red\\\\9;
}
}"
`;

exports[`:ie910 should transform :ie910(...) into an ie910 (single selector) 1`] = `
"@media screen and (min-width:0\\\\0) {.fooga {
    color: red\\\\9;
}
}"
`;

exports[`:ie910 should work with postcss-nested 1`] = `
".fooga {
    color: red;
}
    
    @media screen and (min-width:0\\\\0) {
    
    .fooga {
        color: blue\\\\9;
    }
}"
`;
